docs/architecture/certificates/concepts_and_definitions.rst

```diff
@@ -0,0 +1,32 @@
+Concepts and Definitions
+========================
+
+Partitions
+----------
+Partitions are the attributes associated with a row/record specifying which segment of data they’re part of.
+
+**For example**: a particular record can be associated to User A under Facility B. User A and Facility B will be this record’s partitions.
```

Comment:
Could we more precisely define a partition?

* how many partition(s) a record can be associated with?
* what the implications for being in a partition?